l10n_be_cooperator_national_number/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import test_cooperator_national_number
+
```

Comment:
```suggestion
from . import test_cooperator_national_number
```

Reply:
Don't know why there is a blank line at the end of this file and not at the end of the others.

Reply:
But it's very minor.